serve/mlc_serve/model/tvm_model.py

```diff
@@ -67,36 +132,144 @@ def broadcast_from_worker_0(sess: di.Session, src, shape, dtype):
     return dst
 
 
+def load_tvm_weights(
```

Comment:
This code path seems untested. The function is broken.

Reply:
You are correct, looks like this was a merge error on my part.